lib/langchain/llm/ollama.rb

```diff
@@ -22,6 +22,7 @@ class Ollama < Base
     # @param default_options [Hash] The default options to use
     #
     def initialize(url:, default_options: {})
+      depends_on 'faraday'
```

Comment:
Double quotes, otherwise the linter complains.

Reply:
Hah, I caught that and re-force pushed right afterwards. You were too quick (I was too slow!) ;)